### ISSUE TITLE
Missing key in loading model

### ISSUE THREAD

**Issue by xljh0520** [JiangWenPL/multiperson#42]
Hi, thanks for sharing your code. I run the demo 
```
python3 tools/demo.py --config=configs/smpl/tune.py --image_folder=demo_images/ --output_folder=results/ --ckpt data/checkpoint.pt
```
However, I get this info:
```
unexpected key in source state_dict: fc.weight, fc.bias

missing keys in source state_dict: layer2.1.bn3.num_batches_tracked, layer3.2.bn2.num_batches_tracked, layer2.2.bn2.num_batches_tracked, layer1.0.bn1.num_batches_tracked, layer4.1.bn3.num_batches_tracked, layer1.2.bn2.num_batches_tracked, layer4.0.bn1.num_batches_tracked, layer1.1.bn1.num_batches_tracked, layer1.2.bn1.num_batches_tracked, layer1.1.bn3.num_batches_tracked, layer4.2.bn3.num_batches_tracked, layer4.1.bn2.num_batches_tracked, layer4.0.bn3.num_batches_tracked, layer2.3.bn1.num_batches_tracked, layer3.5.bn3.num_batches_tracked, layer3.3.bn2.num_batches_tracked, layer4.0.downsample.1.num_batches_tracked, layer4.2.bn2.num_batches_tracked, layer2.0.bn1.num_batches_tracked, layer3.4.bn1.num_batches_tracked, layer3.1.bn3.num_batches_tracked, layer3.2.bn1.num_batches_tracked, layer3.3.bn1.num_batches_tracked, layer2.3.bn2.num_batches_tracked, layer3.5.bn2.num_batches_tracked, layer4.2.bn1.num_batches_tracked, layer3.1.bn1.num_batches_tracked, layer1.2.bn3.num_batches_tracked, layer4.1.bn1.num_batches_tracked, layer3.0.bn1.num_batches_tracked, layer2.0.bn3.num_batches_tracked, layer2.1.bn1.num_batches_tracked, layer2.0.bn2.num_batches_tracked, layer2.3.bn3.num_batches_tracked, layer1.0.bn3.num_batches_tracked, layer3.0.downsample.1.num_batches_tracked, layer3.0.bn2.num_batches_tracked, layer3.2.bn3.num_batches_tracked, layer3.4.bn2.num_batches_tracked, layer2.2.bn1.num_batches_tracked, layer3.0.bn3.num_batches_tracked, layer1.0.bn2.num_batches_tracked, layer3.5.bn1.num_batches_tracked, layer1.0.downsample.1.num_batches_tracked, layer2.2.bn3.num_batches_tracked, layer3.1.bn2.num_batches_tracked, layer4.0.bn2.num_batches_tracked, layer2.0.downsample.1.num_batches_tracked, layer3.3.bn3.num_batches_tracked, layer1.1.bn2.num_batches_tracked, bn1.num_batches_tracked, layer2.1.bn2.num_batches_tracked, layer3.4.bn3.num_batches_tracked

2022-08-19 12:51:02,959 - INFO - load checkpoint from data/checkpoint.pt
2022-08-19 12:51:03,424 - WARNING - missing keys in source state_dict: smpl_head.loss.smpl.J_regressor_extra, smpl_head.smpl.v_template, smpl_head.loss.smpl.faces_tensor, smpl_head.loss.smpl.posedirs, smpl_head.smpl.J_regressor_extra, smpl_head.smpl.parents, smpl_head.loss.smpl.lbs_weights, smpl_head.loss.smpl.parents, smpl_head.smpl.shapedirs, smpl_head.smpl.vertex_joint_selector.extra_joints_idxs, smpl_head.smpl.faces_tensor, smpl_head.smpl.posedirs, smpl_head.smpl.lbs_weights, smpl_head.smpl.J_regressor, smpl_head.loss.smpl.vertex_joint_selector.extra_joints_idxs, smpl_head.loss.smpl.v_template, smpl_head.loss.smpl.shapedirs, smpl_head.loss.smpl.J_regressor

2022-08-19 12:51:03,450 - INFO - resumed epoch 27, iter 90901
```
I suppose there are many missing keys so the demo will fail. However, I got a reasonable result. 
![截屏2022-08-19 下午1 00 19](https://user-images.githubusercontent.com/56757835/185546141-2adba51f-be77-40cc-b03b-2667e11fe8f9.png)
I tried as [#1 ](https://github.com/JiangWenPL/multiperson/issues/1). But I still get the same info.

**Comment by JiangWenPL** [comment]
I think these missing keys might be a version issue of mmcv or implementations of batchnorm or optimizers. `num_batches_tracked` should not affect the behaviors during inference.
